proxy/proxy.go

```diff
@@ -231,7 +231,7 @@ func (proxy *proxy) init() error {
 		if err != nil {
 			return fmt.Errorf("couldn't create AF_UNIX socket: %v", err)
 		}
-		if err = os.Chmod(socketPath, 0666|os.ModeSocket); err != nil {
+		if err = os.Chmod(socketPath, 0660|os.ModeSocket); err != nil {
 			return fmt.Errorf("couldn't set mode on socket: %v", err)
```

Comment:
This commit doesn't make it worse than before, so we shouldn't block it on that.  Given that it should only be a debugging path, not terribly critical I'd say.

I'm open to suggestions though!